### PR TITLE
Measure voltage Experiment

### PR DESCRIPTION
**Pull request by NamanRawat1**
Fixes issue #631 
APK:
[pslab.zip.zip](https://github.com/fossasia/pslab-android/files/1313251/pslab.zip.zip)

Screenshots for the change: 
voltage

### CODE REVIEW

**Comment by asitava1998** [comment]
@NamanRawat1  You need to add an apk in the PR description, so that we can try it out.

**Comment by NamanRawat1** [comment]
Github not supporting .apk file.   What to do?

**Comment by asitava1998** [comment]
You have to create a .zip file and then upload it.

File: app/src/main/java/org/fossasia/pslab/experimentsetup/schoollevel/MeasureVoltage.java [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        import android.graphics.Color;
+        import android.os.AsyncTask;
+        import android.os.Bundle;
+        import android.os.Handler;
+        import android.os.Looper;
+        import android.support.annotation.Nullable;
+        import android.support.v4.app.Fragment;
+        import android.view.LayoutInflater;
+        import android.view.View;
+        import android.view.ViewGroup;
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/experimentsetup/schoollevel/MeasureVoltage.java]: Please correct indentation and remove extra lines between code.

**Comment by viveksb007** [comment]
@CloudyPadmal could you look if `AC and DC` experiment and `Measure Voltage` Experiments are same?

**Comment by NamanRawat1** [comment]
Done with reformatting review changes. @viveksb007

**Comment by CloudyPadmal** [comment]
Hi @viveksb007 ! They seem to be! It's just same experiment with different names!

**Comment by asitava1998** [comment]
@viveksb007  Can you review this PR again? Merging is blocked since you requested changes.

**Pull request merged**